go/kbfs/libkbfs/favorites.go

```diff
@@ -300,6 +300,56 @@ func (f *Favorites) closeReq(req *favReq, err error) {
 	}
 }
 
+func (f *Favorites) crossCheckWithEditHistory() {
+	// NOTE: Ideally we would wait until all edit activity processing
+	// had completed in the FBO before we do these checks, but I think
+	// in practice when the mtime changes, it'll be the edit activity
+	// processing that actually kicks off these favorites activity,
+	// and so that particular race won't be an issue.  If we see
+	// weirdness here though, it might be worth revisiting that
+	// assumption.
+
+	// Fix the favorite times to be the latest known edit history
+	// times, if possible.  If not possible, set them to be lower than
+	// the minimum known time, if they're not already.
+	var minTime keybase1.Time
+	uh := f.config.UserHistory()
+	tlfsWithNoHistory := make(map[favorites.Folder]favorites.Data)
+	for fav, data := range f.favCache {
+		h := uh.GetTlfHistory(tlf.CanonicalName(fav.Name), fav.Type)
+		if h.ServerTime == 0 {
+			if data.TlfMtime != nil {
+				tlfsWithNoHistory[fav] = data
+			}
+			continue
+		}
+		if minTime == 0 || h.ServerTime < minTime {
+			minTime = h.ServerTime
+		}
+		// Seems like the mtime in the favorites list is incorrectly
```

Comment:
maybe we should fix this? If you don't want to because it includes frontend code, make a ticket and song or I could take a whack at it?

Reply:
@heronhaye said he'd fix this, so I'll hold off on merging this until that's in.

Reply:
Should be fixed in kbweb master, let me know if not